cms/static/js/certificates/views/signatory_editor.js

```diff
@@ -178,6 +183,29 @@ function ($, _, Backbone, gettext,
                 }
             });
             modal.show();
+        },
+
+        toggleValidationErrorMessage: function(modelAttribute) {
+            // Toggle the validation error messages.
+            // If given model attribute is not valid then show the error message else remove it.
+            if (!this.model.isValid() && _.has(this.model.validationError, modelAttribute)) {
+                addErrorMessage(this.model.validationError[modelAttribute]);
+            } else {
+                removeErrorMessage();
+            }
+
+            function addErrorMessage (errorMessage) {
+                // adding the error message into the dom element.
+                $("div.add-signatory-" + modelAttribute).addClass("error");
+                $("div.add-signatory-" + modelAttribute).append(
+                        "<span class='message-error'>" + errorMessage + "</span>");
+            }
+
+            function removeErrorMessage () {
+                // removing the error message from the dom element.
+                $("div.add-signatory-" + modelAttribute).removeClass("error");
+                $("div.add-signatory-" + modelAttribute + ">span.message-error").empty();
```

Comment:
I think you should remove the message-error `span` instead of emptying it. With current implementation it would create multiple span nodes as if `toggleValidationErrorMessage` is called multiple time for a model attribute.